pkg/attestation/crafter/crafter.go

```diff
@@ -427,6 +431,15 @@ func (c *Crafter) ResolveEnvVars(ctx context.Context, attestationID string) erro
 	return nil
 }
 
+func (c *Crafter) ResolveHostedRunnerContext(ctx context.Context) {
+	c.Logger.Debug().Bool("isHosted", c.Runner.IsHosted(ctx)).Msg("is runner hosted")
+	c.CraftingState.Attestation.IsHostedRunner = c.Runner.IsHosted(ctx)
+	c.Logger.Debug().Bool("isHosted", c.Runner.IsHosted(ctx)).Msg("is runner authenticated")
```

Comment:
this log entry is strange, it seems to be talking both about is_hosted and authenticated

Reply:
Yeah, copy+paste mistake, thanks for noticing. Fixed.